docs/requirements.txt

```diff
@@ -0,0 +1 @@
+git+git://github.com/ggbecker/sphinxcontrib.jinjadomain.git#egg=sphinxcontrib-jinjadomain
```

Comment:
Unfortunately the official package is not maintained anymore and it was not working. I had to fork and fix some issues to adapt for our purposes.

Reply:
That's a different module, it allows to use Jinja macros in rst docs

Reply:
@jan-cerny is correct. The package we are using serves the purpose of processing comments on our jinja macros so they can be nicely displayed in the documentation, think of something similar to python docstrings but for jinja macros.